internal/scorecard/scorecard.go

```diff
@@ -111,9 +112,12 @@ func (s Config) RunTests() error {
 	return nil
 }
 
+var lastSDKVersionWithDocDir = semver.MustParse("0.16.0")
+
 func ConfigDocLink() string {
-	if strings.HasSuffix(version.Version, "+git") {
-		return "https://github.com/operator-framework/operator-sdk/blob/master/doc/test-framework/scorecard.md"
+	ver, err := semver.ParseTolerant(version.Version)
+	if strings.HasSuffix(version.Version, "+git") || err != nil || ver.GT(lastSDKVersionWithDocDir) {
+		return "https://sdk.operatorframework.io/docs/scorecard/"
```

Comment:
it ould be an util centralized. shows that other places could use it.

Reply:
Actually this should always return the SDK website.